layouts/_partials/recent-resources.html

```diff
@@ -4,17 +4,14 @@
       <h2 class="mb-8 mt-16 text-4xl font-extrabold">{{ i18n "featured" }}</h2>
       <p class="mt-2 text-lg/8 text-gray-600">{{ i18n "resources-description" }}</p>
     </div>
-    {{ if .Site.Params.homepage.showRecent | default false }}
-      <!-- <h2 class="mt-16 mb-8 text-4xl font-extrabold">{{ i18n "featured" | emojify }}</h2> -->
```

Comment:
Just these two lines being deleted. I doubt we'll ever turn off showRecent on the homepage, so might aswell always include it.